mayastor/src/bdev/nexus/nexus_bdev.rs

```diff
@@ -1080,12 +1100,40 @@ pub async fn nexus_create(
         return Ok(());
     }
 
+    let (min_cntlid, max_cntlid, resv_key) =
```

Comment:
This validation could be done then as an impl on the struct rather than adding this